massa-bootstrap/src/server.rs

```diff
@@ -224,23 +227,18 @@ impl BootstrapServer<'_> {
         interval: Duration,
         stopper: Receiver<()>,
     ) -> Result<(), Box<BootstrapError>> {
+        let ticker = tick(interval);
+
         loop {
-            std::thread::sleep(interval);
-            match stopper.try_recv() {
-                Ok(()) => return Ok(()),
-                Err(e) => match e {
-                    crossbeam::channel::TryRecvError::Empty => {}
-                    crossbeam::channel::TryRecvError::Disconnected => {
-                        return Err(BootstrapError::GeneralError(
-                            "update stopper unexpected disconnect".to_string(),
-                        )
-                        .into());
+            select! {
```

Comment:
```suggestion
            crossbeam::select! {
```
Similar in principal to the thread comment